### DISCUSSION TITLE
[SOLVED] After importing a 3D model and setting the boundary to itself, the model always bounces

### DISCUSSION THREAD

**Discussion by lecheng532** [cyberbotics/webots#6517] [General]
I built the model using solidworks software, imported it into webots2023, set the border to itself, and found that it kept bouncing, but set the border to box and didn't bounce. But I want the boundary to be this model and not box. What should I do？

**Comment by omichel** [comment]
You should create a compound bounding object, made-up of several primitives (boxes, spheres, capsules, spheres), each of them encapsulated inside a Pose node for orientation and position and all the Pose nodes put together inside a Group node for the boundingObject field of the Solid node. See tutorial here: https://cyberbotics.com/doc/guide/tutorial-5-compound-solid-and-physics-attributes

**Reply by lecheng532** [reply]
Your advice is correct. Thank you very much
